static/docs/user-guide/dvcignore.md

```diff
@@ -18,21 +17,22 @@ provide similar functionality as `.gitignore` files provide for `git`.
 - Each line should contain only one pattern.
 - During execution of commands that traverse directories, DVC will ignore
   matching paths.
-- Not every operation supports `.dvcignore`. To see current limitations, read
-  following paragraph.
-
-## Current limitations
-
-During development, we noticed that there are few potential uses cases that
-might be tricky to handle (e.g. what to do when we are `dvc add`-ing directory
-containing `.dvcignore` file). Therefore, we decided to enable this feature
-gradually in different parts of the project.
-
-Currently `.dvcignore` files will be read and applied in any operation that
-collects DVC-files (e.g. `checkout`, `metrics`, `status`, `run`, `repro`), so it
-is advised to use it in cases described in the first paragraph, when amount of
-files in tree of repository directory causes performance issues.
 
+## Notes
+
+- `.dvcignore` is no longer applied only on operations collecting DVC-files. 
```

Comment:
no need to mention this. No need to reference how was it before. We need to describe the current state.